packages/honeycomb-opentelemetry-web/rollup.config.mjs

```diff
@@ -1,33 +1,17 @@
-import { createRequire } from 'node:module';
 import { DEFAULT_EXTENSIONS } from '@babel/core';
 import analyze from 'rollup-plugin-analyzer';
-import babel from '@rollup/plugin-babel';
+import { babel } from '@rollup/plugin-babel';
 import commonjs from '@rollup/plugin-commonjs';
 import dts from 'rollup-plugin-dts';
-import nodeResolve from '@rollup/plugin-node-resolve';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
```

Comment:
Why did these imports change? It looks like the package versions stayed the same...

Reply:
`@rollup/plugin-node-resolve` and `@rollup/plugin-babel` both have names export AND exports that same thing as the default export. It was a linting violation to alias the default export the same as named export, so we just use the named export instead.